Day3Part2.js

```diff
@@ -0,0 +1,53 @@
+const readline = require("readline").createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+readline.question(`enter your data `, function LifeSupportRating(data) {
+  var Data = data.split(" ");

```

Comment:
There should be a better name than "data"

```suggestion
  const dataArray = data.split(" ");

```